docs/deployment/kernel-tuning/README.md

```diff
@@ -50,6 +50,14 @@ sysctl -w vm.dirty_background_ratio=1
 sysctl -w vm.dirty_ratio=5
 ```
 
+- *`Transparent Hugepage Support`*: This is a Linux kernel feature intended to improve performance by making more efficient use of processor’s memory-mapping hardware. But this may cause [problems](https://blogs.oracle.com/linux/performance-issues-with-transparent-huge-pages-thp) for non optimized applications. As most Linux distributions set it to `enabled=always` by default, we recommend changing this to `enabled=madvise`. This will allow applications optimized for transparent hugepages to obtain the performance benefits, while preventing the associated problems otherwise.
```

Comment:
``` for non optimized applications ``` to ``` for non-optimized applications ```

``` optimized for transparent hugepages to obtain the performance benefits, ``` to ``` optimized for transparent huge pages to obtain the performance benefits ```

Reply:
> optimized for transparent huge pages to obtain the performance benefits

The feature is referred to as _hugepage_, almost all mentions [here](https://www.kernel.org/doc/Documentation/vm/transhuge.txt) are _hugepage_

Reply:
> for non optimized applications to for non-optimized applications

Done